apis/apis.json

```diff
@@ -54,9 +59,13 @@
     "type": "grpc",
     "description": "Recommended Google client library to access the Google Cloud Datastore API, which accesses the schemaless NoSQL database to provide fully managed, robust, scalable storage for your application.",
     "tags": [ "Datastore" ],
+    "dependencies": {
+      "Grpc.Core": "1.4.0",
+      "Google.Api.Gax.Grpc": "2.0.0"
```

Comment:
I don't really like this requirement to hardcode versions here. It seems to somewhat defeat the point of generating project files. As more packages go GA it's going to get more difficult to update all these correctly.